File: API/Python 3.4/Python1/AI.py
import game_api
import json

class AI:
    def __init__(self, conn):
        game_api.SOCKET_CONN = conn
        #Replace by your team name!
        self.name = 'BOT'
        
    def update(self, game_objects):
        #do stuff here
        print('Updating')
        
    def set_name(self):
        print('Setting Name')
        game_api.set_name(self.name)

File: API/Node.js/Node1/AI.js
// Set your team name here!!!
var name = "Bot";
var game;

var set_game = function (obj) {
  game = obj;
}

var update = function (game_objects) {
  console.log("update");
};

var set_name = function () {
  console.log("set_name");
  game.set_name(name);
};

module.exports = {
  set_name: set_name,
  update: update,
  set_game: set_game
};